borbskeeper/Features/TaskList/TasksListViewController.m

```diff
@@ -80,10 +80,10 @@ - (void)fetchData {
         [self.taskListInfiniteScrollView.tableView reloadData];
         [self.activityIndicator stopAnimating];
         [self.refreshControl endRefreshing];
-        [self checkDate];
+        //[self checkDate];
```

Comment:
is this intended? Let's remove if it is instead of commenting

Reply:
That was code written by someone else, but it was breaking my application; I commented it out then so I could run without deleting someone else's work. I've talked to the team since and they said it was okay to delete, so I did.